testing/sakura/APKBUILD

```diff
@@ -0,0 +1,29 @@
+# Contributor: Niko Dittmann <mail@niko-dittmann.com>
+# Maintainer: Niko Dittmann <mail@niko-dittmann.com>
+pkgname=sakura
+pkgver="3.5.0"
+pkgrel=0
+pkgdesc="Sakura Terminal Emulator"
+url="https://launchpad.net/sakura"
+arch="all"
+license="GPL-2.0"
+depends="vte3 pango gtk+3.0 glib libintl"
+makedepends="cmake vte3-dev pango-dev gtk+3.0-dev glib-dev libintl"
+install=""
+subpackages="$pkgname-doc"
+source="https://launchpad.net/sakura/trunk/$pkgver/+download/$pkgname-$pkgver.tar.bz2"
+builddir="$srcdir/"
```

Comment:
Default path for builddir is `"$srcdir"/$pkgname-$pkgver`, so just remove `builddir="$srcdir/"`